python/tests/io/hdf5_test.py

```diff
@@ -151,13 +151,6 @@ def test_data_array_dtype_scipp_container():
     a = sc.DataArray(data=x)
     a.coords['variable'] = sc.scalar(x)
     a.coords['scalar'] = sc.scalar(a)
-    a.coords['1d'] = sc.Variable(dims=a.dims,
-                                 shape=a.shape,
-                                 dtype=sc.dtype.DataArray)
-    a.coords['1d'].values[0] = sc.DataArray(data=0.0 * sc.units.m)
-    a.coords['1d'].values[1] = sc.DataArray(data=1.0 * sc.units.m)
-    a.coords['1d'].values[2] = sc.DataArray(data=2.0 * sc.units.m)
-    a.coords['1d'].values[3] = sc.DataArray(data=3.0 * sc.units.m)
```

Comment:
Why did you remove this part of the test?

Reply:
Because there is no straight forward way of making such a variable anymore.

Reply:
That is not a valid reason for removing an important test.

You can make such a variable using `concatenate`.

Reply:
`broadcast` should also work, instead of `concatenate`.